room/room-compiler-processing/src/main/java/androidx/room/compiler/processing/ksp/KspProcessingEnv.kt

```diff
@@ -41,11 +41,18 @@ internal class KspProcessingEnv(
     override val options: Map<String, String>,
     codeGenerator: CodeGenerator,
     logger: KSPLogger,
-    val resolver: Resolver
 ) : XProcessingEnv {
     override val backend: XProcessingEnv.Backend = XProcessingEnv.Backend.KSP
     private val ksFileMemberContainers = mutableMapOf<KSFile, KspFileMemberContainer>()
 
+    private var _resolver: Resolver? = null
+
+    var resolver
+        get() = _resolver!!
+        set(value) {
```

Comment:
Make the setter `internal`:
`internal set(value) {`

Reply:
Done.